fedimint-core/src/config.rs

```diff
@@ -155,6 +156,8 @@ pub struct GlobalClientConfig {
     /// API endpoints for each federation member
     #[serde(deserialize_with = "de_int_key")]
     pub api_endpoints: BTreeMap<PeerId, PeerUrl>,
+    /// Signing session keys for each federation member
+    pub broadcast_public_keys: Option<BTreeMap<PeerId, PublicKey>>,
```

Comment:
Please add comment explaining the `Option`

Reply:
With the changes to calculate_federation_id it does test both

Reply:
No comment explaining that this field was added and might not be available on old clients.